website/docs/r/cosmosdb_sql_container.html.markdown

```diff
@@ -96,6 +98,14 @@ An `excluded_path` block supports the following:
 
 * `path` - Path that is excluded from indexing.
 
+A `composite_index` block supports the following:
+
+* `index` - One or more `index` blocks as defined below.
+
+An `index` block supports the following:
+
+* `path` - Path for which the indexing behavior applies to.
+* `order` - Order of the index. Can be either `ascending` or `descending`.
```

Comment:
these are cased "Ascending" and "Descending" in the SDK - so can we make this:

```suggestion
* `order` - Order of the index. Possible values are `Ascending` or `Descending`.
```

(which also changes the text slightly to match other resources)